### DISCUSSION TITLE
Graph view?

### DISCUSSION THREAD

**Discussion by arcanero-nick** [zotonic/zotonic#3764] [Q&A]
Hi all, Zotonic newbie here.  

I'm looking for a platform to develop a PERT-style web application for large projects, using resources and edges for milestones and tasks.  I'm already an Erlang fan, even though I'm yet to use it in a project.  I was wondering if there was already a graph view of the Zotonic topic list, or whether that's the first thing I'd need to develop.

Cheers and Thanks

**Comment by geoidesic** [comment]
What do you mean by "graph view"? Can you attach an image of an example?

**Comment by mworrell** [comment]
In mod_development we have a graph view for the included templates.
For that we generate a "dot" file and let viz.js show it.

I know that Driebit has also a kind of dynamic relation thing made, for example:

https://openresearch.amsterdam/en/page/114044/doorgroeibare-en-waterdoorlatende-verhardingen-tegels#graph-section
